pytest_tags/pytest_tags.py

```diff
@@ -25,7 +25,8 @@ def pytest_configure(config):
 
     # Create tags container based on command line parameters
     browser = None
-    if hasattr(config.option, 'driver'):
+    driver = getattr(config.option, 'driver', None)
```

Comment:
here's my suggested solution: `if getattr(config.option, 'driver', False)`
The reason being that `driver` is not used if it doesn't exist or is set to `None`

Reply:
That's right. I have made the necessary changes now.